src/components/ContactsApp.jsx

```diff
@@ -63,33 +49,19 @@ class ContactsApp extends React.Component {
 
     return (
       <main className="app-content">
-        <ContactsHeader />
+        <ContactsHeaderWithActions />
```

Comment:
It looks like it's not a big change, but it add disempowerment to the `<ContactsHeader />` component.